docs/docs/propeller/configuring.md

```diff
@@ -48,3 +48,10 @@ that idempotent handling is a requirement, checkpointing is an optimisation and
 your system should behave the same whether it starts from scratch on every start
 or at the last stored checkpointed.
 
+This last point is important, in practise it is exceedingly rare for your system
```

Comment:
Added this section to make sure we get the point across that "this is not some tinfoil corner case"